src/molecules/optionSelect/component.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import { Box, HidingView, Text } from '../..';
+import { IBoxTheme } from '../../particles';
+import { IListTheme, List } from '../list';
+import { defaultMoleculeProps, IMoleculeProps } from '../moleculeProps';
+import { ISingleLineInputTheme, SingleLineInput } from '../singleLineInput';
+
+interface IOption {
+  text: string;
+  itemKey: string;
+  isDisabled?: boolean;
+}
+
+export interface IOptionSelectTheme {
+  // (I think so) Need to write a buildTheme file for OptionSelect, to give default style the components
+  // or should I be using styled components
+  inputHandler: ISingleLineInputTheme;
+  optionsContainer: IBoxTheme;
+  optionList: IListTheme;
+}
```

Comment:
Need help with this

Reply:
look at how singlelineinput has a theme, i think this will be similar